example.env

```diff
@@ -9,6 +9,7 @@
 # See the "nox -s test_env" command or scripts/setup/load_examples.py for
 # details on how these are used.
 #
+FIDES__CONFIG_PATH=/fides/data/config/fides.test_env.toml
```

Comment:
On reflection, I feel like this probably doesn't work. It relies on users to delete their `.env` file and create a new one from the example template again...

I should find a better way to inject this env variable into the nox command, it's just so awkward to modify that `START_APP_EXTERNAL` constant...

Reply:
i'm sure we could figure out a more elegant paradigm for injecting env vars....

I would actually be a fan of a pure env var solution, but maybe I'm just a bit scarred from putting secrets anywhere now 😬

Reply:
as long as we're documenting how to run things....